docs/rules/html-attributes-casing.md

```diff
@@ -0,0 +1,35 @@
+# Define a style for the attributes casing in templates. (html-attributes-casing)
```

Comment:
Thank you for the contributing! I'm sorry for the delay.

I'm not sure whether this rule makes sense. Below is the result of my investigation.

- In all known HTML/SVG/MathML elements, all of `attrname`, `attrName` and `AttrName` are the same attribute but `attr-name` is another.
- In custom components,
    - about defined properties,
        - for example `attrName` property, people can choose `attrName` or `attr-name` but `attrname` and `AttrName` do not work.
        - for example `AttrName` property, people can choose `AttrName` or `attr-name` but `attrname` and `attrName` do not work.
    - about undefined properties, the same behaivor to known HTML/SVG/MathML elements.

So I think the options `camelCase` and `PascalCase` are not useful because it might be uncontrollable due to the definition of libraries. So maybe.... What do you think below?

- Change rule name to `attribute-hyphenation`
- Options are: `"always"` (default) or `"never"`.
    - `"always"` ... Use hyphenated name.
    - `"never"` ... Don't use hyphenated name.

(I think this rule don't need `html-` prefix because this rule can work on AST. I think that rules which use details of tokens should have `html-` prefix.)